### PR TITLE
Test all const are in uppercase

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
nice idea

File: tests/AutoReview/ProjectCodeTest.php [hunk 1]
```diff
@@ -742,6 +746,22 @@ public function providePhpUnitFixerExtendsAbstractPhpUnitFixerCases()
         }
     }
 
+    /**
+     * @param string $className
+     *
+     * @dataProvider provideSrcClassCases
```
Comment (keradus) [inline tests/AutoReview/ProjectCodeTest.php:752]: maybe we have some constants in test classes too?

File: tests/AutoReview/ProjectCodeTest.php [hunk 2]
```diff
@@ -742,6 +746,22 @@ public function providePhpUnitFixerExtendsAbstractPhpUnitFixerCases()
...
+     * @requires PHP 7.1
```
[the remark this answers is not in the document]
Comment (SpacePossum) [reply tests/AutoReview/ProjectCodeTest.php:753]: ah yes, can [7.0](https://www.php.net/manual/en/reflectionclass.getreflectionconstants.php)

[the remark this answers is not in the document]
Comment (SpacePossum) [reply tests/AutoReview/ProjectCodeTest.php]: hmm guess not, https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5750/checks?check_run_id=2692838774  so yeah 7.1 it is, we only need the test on one PHP version anyhow

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**